src/js/containers/Home.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+// Imported Components
+import Post from '../components/Post';
+
+// Imported Actions
+import * as actions from '../actions/index';
+
+const mapStateToProps = (state) => {
+  return {
+    posts: state.posts
+  };
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return bindActionCreators(actions, dispatch);
+};
+
+
+class Home extends React.Component {
+  render() {
+    // Post Loop
+    const posts = this.props.posts.map((post, index) => <Post key={index} index={index} post={post} />);
+
+    return (
+      <div>
```

Comment:
Yes. Although now that I am looking, does this actually work? The HTML attribute to control CSS classes is `class` such as `<div class="row">`.